docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -229,6 +229,26 @@ To upgrade legacy multi-cluster apps:
 1. Click **☰**. 
 1. Under **Legacy Apps**, click **Multi-cluster Apps**.
 
+### Chart-Specific Information
+
+#### sriov Chart Deprecation and Migration
+
+The sriov (SR-IOV network operator) chart from the Rancher Charts repository is deprecated and will be removed in Rancher v2.10. Please migrate to the sriov-network-operator chart from the SUSE Edge repository (https://github.com/suse-edge/charts) instead.
```

Comment:
Should this be `sriov` (formatted with backticks)?

Reply:
Formatting added. Was undecided on whether to treat the chart name as a proper noun or not.